sdks/python/README.md

```diff
@@ -14,6 +14,33 @@ To install the latest development version of the SDK, run the following:
 pip install git+https://github.com/argoproj/argo-workflows@master#subdirectory=sdks/python/client
 ```
 
+## Contributing
```

Comment:
it is not possible for users to tag, that can only be done by whomever is released, and tags are `git tag v3.2.5` with `v` prefix